src/components/a-device-state/_preview.html

```diff
@@ -1 +1 @@
-<div class="a-device-state"></div>
+<div class="a-device-state js-device-state"></div>
```

Comment:
Remember we talked about moving it to `utility`, what about doing it right now?

Reply:
is not a utility yet as css is loaded as a component. We need to engeener it better later one. im in such a stress at the moment that i would prefer to keep this for later on